src/SSRv1/templates/newItem.php

```diff
@@ -30,14 +32,19 @@
 		<?php if(isset($base) && $base->hasCode()): ?>
 			<div class="inline-alert alert-info" role="alert"><i class="fa fa-info-circle"></i>️&nbsp;This is a copy of <span class="text-monospace"><?= $base->getCode() ?></span>, remember to change serial numbers, notes, working status, etc...</div>
 		<?php unset($noticeFeature); endif; ?>
+
+		<?php if(isset($importedFrom)): ?>
+			<div class="inline-alert alert-info" role="alert"><i class="fa fa-info-circle"></i>️&nbsp;This item is generated from a bulk import ( with Id = <span class="text-monospace"><?= $importedFrom ?> ) </span></div>
+			<?php unset($noticeFeature); endif; ?>
 	</header>
 
 	<nav class="itembuttons row mx-2 mt-2 justify-content-end">
 		<?php if(!$innerrecursion): ?>
 			<button class="btn btn-outline-primary btn-item col-4 col-sm-auto mr-auto cancel" role="button">
 				<i class="fa fa-arrow-circle-left"></i>&nbsp;Cancel
 			</button>
-			<button class="btn btn-outline-primary btn-item col-4 col-sm-auto save" role="button">
+			<button class="btn btn-outline-primary btn-item col-4 col-sm-auto save" role="button"
```

Comment:
Interesting way to do it asd

I'd still suggest using `data-BulkId="..."` (read it with document._get.to.that.element_.dataset.BulkId, it's a variable) or something.